queue/queue.test.js

```diff
@@ -0,0 +1,91 @@
+const Queue = require("./Queue");
+
+test('Queue size should be zero with out adding any element', () => {
+    var queue = new Queue(5);
+
+    expect(queue.count()).toBe(0);
+});
+
+test('Queue size should be one after adding 1 element', () => {
+    var queue = new Queue(5);
+    queue.enqueue(11);
+    expect(queue.count()).toBe(1);
+});
+
+
+test('count() should match the number of elements added', () => {
+    var queue = new Queue(5);
+    [11, 12, 13, 14].forEach(elem => queue.enqueue(elem));
+    expect(queue.count()).toBe(4);
+});
+
+test('count() should reduce every time dequeue() is called', () => {
+    var queue = new Queue(5);
+    [11, 12, 13, 14, 15].forEach(elem => queue.enqueue(elem));
+    expect(queue.count()).toBe(5);
```

Comment:
also, can we loop line 25 and 26? This will make this test readable in the same sense as it did on line 18